.clang-format

```diff
@@ -3,7 +3,7 @@ BasedOnStyle:  LLVM
 BreakConstructorInitializersBeforeComma: true
 ConstructorInitializerAllOnOneLineOrOnePerLine: true
 Cpp11BracedListStyle: true
-Standard: Cpp11
+Standard: c++20
```

Comment:
Did the way to specify standards change?

Reply:
TIL https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard

Reply:
TIL https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard

Also, LOL "Cpp11 is a deprecated alias for Latest".

Reply:
I just copy what you did for epic